tools/js-sdk-release-tools/src/hlc/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -62,18 +69,34 @@ function changeContextFile(packageFolderPath: string, packageVersion: string) {
     })
 }
 
+function changeClientFile(packageFolderPath: string, packageVersion: string) {
+    const packageJsonData: any = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'package.json'), 'utf8'));
+    const packageName = packageJsonData.name.replace("@azure/", "");
+    const files: string[] = fs.readdirSync(path.join(packageFolderPath, 'src'));
+    files.forEach(file => {
+        if (file.endsWith('Client.ts')) {
+            const data: string = fs.readFileSync(path.join(packageFolderPath, 'src', file), 'utf8');
+            const result = data.replace(/const packageDetails = `azsdk-js-[0-9a-z-]+\/[0-9.a-z-]+`;/g, 'const packageDetails = `azsdk-js-' + packageName + '/' + packageVersion + '`;');
+            fs.writeFileSync(path.join(packageFolderPath, 'src', file), result, 'utf8');
+        }
+    })
+}
+
 export function makeChangesForReleasingTrack2(packageFolderPath: string, packageVersion: string, changeLog: Changelog) {
     const originalChangeLogContent = fs.readFileSync(path.join(packageFolderPath, 'changelog-temp', 'package', 'CHANGELOG.md'), {encoding: 'utf-8'});
-    const modifiedChangelogContent = `## ${packageVersion} (${date})
+    const modifiedChangelogContent = `# Release History
+    
+## ${packageVersion} (${date})
     
 ${changeLog.displayChangeLog()}
     
-${originalChangeLogContent}`;
+${originalChangeLogContent.replace(/.*Release History[\n\r]*/g, '')}`;
 
     fs.writeFileSync(path.join(packageFolderPath, 'CHANGELOG.md'), modifiedChangelogContent, {encoding: 'utf-8'});
 
     changePackageJSON(packageFolderPath, packageVersion);
-    changeContextFile(packageFolderPath, packageVersion)
+    changeContextFile(packageFolderPath, packageVersion);
```

Comment:
I don't think we need this in the latest code generator ?

Reply:
Yes, but I think it has not been released. So keep it, and it will do nothing if there is not Context file